File: app/src/main/java/org/runnerup/features/SettingsUnitsFragment.java
package org.runnerup.features;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;
import org.runnerup.R;

public class SettingsUnitsFragment extends PreferenceFragmentCompat {

  @Override
  public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
    setPreferencesFromResource(R.xml.settings_units, rootKey);
  }
}

File: app/src/main/kotlin/org/runnerup/features/SettingsUnitsFragment.kt
package org.runnerup.features

import android.os.Bundle
import org.runnerup.R

class SettingsUnitsFragment : SettingsFragment() {

  override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
    setPreferencesFromResource(R.xml.settings_units, rootKey)
  }
}
